package.json

```diff
@@ -104,22 +105,22 @@
 		"postcss-extend": "^1.0.5",
 		"postcss-hover-media-feature": "^1.0.2",
 		"postcss-import": "^16.1.0",
-		"postcss-licensing": "^2.0.0",
-		"postcss-load-config": "^5.1.0",
+		"postcss-licensing": "^3.0.0",
+		"postcss-load-config": "^6.0.1",
 		"postcss-modules": "^6.0.1",
-		"postcss-preset-env": "^9.5.13",
+		"postcss-preset-env": "^10.1.5",
```

Comment:
This is the primary update in the PR as results in the change to the focus-within selectors.